stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -258,6 +258,7 @@ internal class PaymentSheetViewModel internal constructor(
             else -> {
                 eventReporter.onPaymentFailure(PaymentSelection.GooglePay)
 
+                paymentIntent.value?.let(::resetViewState)
                 // TODO(mshafrir-stripe): handle error
```

Comment:
can we resolve this comment now?

Reply:
No, this handles cancellation but we still have to go through other possible errors. We're tracking that separately.